>From fc2aeb4d5915807a4ab8b62ff5adf3d1ead3c01d Mon Sep 17 00:00:00 2001
From: Adam Borowski <kilobyte@angband.pl>
Date: Wed, 16 Oct 2013 19:13:58 +0200
Subject: [PATCH] Fix unrand tile mismatches between architectures.

They were written in hash order, which is not supposed to be stable.
---
 crawl-ref/source/util/art-data.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crawl-ref/source/util/art-data.pl b/crawl-ref/source/util/art-data.pl
index a7ce797..5093ca9 100755
--- a/crawl-ref/source/util/art-data.pl
+++ b/crawl-ref/source/util/art-data.pl
@@ -775,7 +775,7 @@ sub write_tiles
 HEADER_END
 
     # Output the tile definitions sorted by type (and thus path).
-    foreach my $type (keys %art_by_type)
+    foreach my $type (sort keys %art_by_type)
     {
         print TILES "%sdir item/$type/artefact\n";
 
-- 
1.8.5.rc0

